### PR TITLE
fix: Prevent celery from printing out vegetables to the terminal

### PR DESCRIPTION
**Pull request by untitaker**
image

when it works it's ok, when it doesn't it's base64

### CODE REVIEW

**Review by markstory — APPROVED** [review]
🥦

File: src/sentry/monkey/__init__.py [hunk 1]
```diff
@@ -84,5 +84,18 @@ def patch_django_views_debug():
     debug.get_safe_settings = lambda: {}
 
 
-for patch in (patch_parse_cookie, patch_httprequest_repr, patch_django_views_debug):
+def patch_celery_imgcat():
+    # Remove Celery's attempt to display an rgb image in iTerm 2, as that
+    # attempt just prints out base64 trash in tmux.
+    from celery.utils import term
+
+    term.imgcat = lambda *a, **kw: b""
```
Comment (joshuarli) [inline src/sentry/monkey/__init__.py:92]: lol

**Comment by mattrobenolt** [comment]
lol what

**Review by dashed — APPROVED** [review]
🥕

**Comment by mattrobenolt** [comment]
You’ll need to try...except ImportError because this gets executed in setup.py before things are installed. The other patches do the same.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**